node/resolvers/catalog/index.ts

```diff
@@ -255,16 +361,36 @@ export const queries = {
         ) as any
       }
 
-      response.category = found && found.id as any
+      response.category = found && (found.id as any)
     }
 
     return response
   },
 }
 
+interface Brand {
+  id: string
+  name: string
+  isActive: boolean
+  title: string
+  metaTagDescription: string
+}
+
 interface Category {
   id: string
   name: string
   url: string
   children: Category[]
+  hasChildren: boolean
+  Title: string
+  MetaTagDescription: string
+}
+
+interface Collection {
+  id: number
```

Comment:
```suggestion
  id: ID
```

Reply:
This is a TypeScript interface, not a GraphQL type.